src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,24 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] stringArray = text.split("[ ]");
+        List<String> numbers = new ArrayList<>();
+        for (String s : stringArray) {
+            numbers.add(s.replaceAll("[^-0-9]", " "));
+        }
+        List<Integer> list = new ArrayList<Integer>();
+        for (String s : numbers) {
```

Comment:
this solution is too complicated, try to solve this task using one for loop and you need to call ".replaceAll()) only once 
hint, you can split your string like this: variables .replaceAll("your regex", " ").trim().split(" ")